src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/page/EditIssuePage.kt

```diff
@@ -49,4 +53,52 @@ class EditIssuePage(
             webElement.sendKeys(text)
         }
     }
+
+}
+
+object PageRteUtil{
```

Comment:
So this is a Kotlin `object`, which is basically a singleton, which is basically a global.
Instead of using a global container for functions, we can keep things local via actual object instances. We can follow the page object pattern, just like we do with `IssueCreateDialog`, `TextArea` etc. The Jira editor is just another page component.

Reply:
yes, agree